binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -128,6 +131,14 @@ public void run() {
                                 }
                             }
                         }
+                        
+                        final Config config = fullConfig.getConfig(); 
+                        if (config != null) {
+                            getThing().getConfiguration().put(ThingProperty.SERIAL_NUMBER.keyName,
```

Comment:
But what happens if the firmware version has been changed? My suggestion is that the properties are only filled / updated here if they have not been specified yet or if they have been changed.... ok?